### ISSUE TITLE
[Backport - 2.3.9] Add istio 1.4.9

### ISSUE THREAD

**Issue by rancher-max** [rancher/rancher#27222]
Backport https://github.com/rancher/rancher/issues/27064 - Need to complete all testing for Istio v1.4.9 in 2.3.x release

**Comment by rancher-max** [comment]
#### Validated in v2.3-head using system-charts branch dev-v2.3

- Istio v1.4.900 is available in Rancher and deploys Istio v1.4.9 correctly
- All additional link pages show up correctly
- Embedded charts on Istio > Traffic Metrics and Traffic Graph show correct data
- Can install/upgrade/modify/delete through UI
- All images deployed are correct 

Airgap testing will be validated as part of #27064 and does not need to be validated again here.

**Comment by deniseschannon** [comment]
Reopened as it's not going out with v2.3.8 and min/max should be reviewed for v2.3.x

**Comment by cbron** [comment]
@deniseschannon the min/max is the same for all branches:

1.4.7
```
rancher_min_version: 2.3.4-rc1
rancher_max_version: 2.3.6
```

1.4.9
```
rancher_min_version: 2.3.7-rc1
```

**Comment by deniseschannon** [comment]
It will need to be updated/changed to:


1.4.7
```
rancher_min_version: 2.3.4-rc1
rancher_max_version: 2.3.7
```

1.4.9
```
rancher_min_version: 2.3.8-rc1
```

**Comment by rancher-max** [comment]
#### Validated min/max behavior is correct on v2.3.7 and v2.3.8 Rancher

- on Rancher v2.3.7, pointing to system-charts branch dev-v2.3, the only available Istio version is v1.4.700
- on Rancher v2.3.8, pointing to system-charts branch dev-v2.3, the only available Istio version is v1.4.900
- In both above cases, Istio installs and runs correctly and uses the expected image versions.
